events/hfe_events_discovery.txt

```diff
@@ -122,6 +150,7 @@ country_event = {
 					}
 				}
 				set_leader = last_created_leader
+				save_global_event_target_as = hfe_discovery_science
```

Comment:
Overusing global event targets is not good

Reply:
Wrong. Wrong. Wrong.

I have made use of countless Global Event Targets. Removing even one would crash the entire Mod. Few things have helped me more.